install-deps.sh

```diff
@@ -345,7 +345,8 @@ else
     centos|fedora|rhel|ol|virtuozzo)
         yumdnf="yum"
         builddepcmd="yum-builddep -y --setopt=*.skip_if_unavailable=true"
-        if test "$(echo "$VERSION_ID >= 22" | bc)" -ne 0; then
+        if test "$(echo "$VERSION_ID >= 22" | bc)" -ne 0 ||
+           ( ( [ "$ID" = "centos" ] || [ "$ID" = "rhel" ] ) && test "$(echo "$VERSION_ID >= 8" | bc)" -ne 0 ); then
```

Comment:
I guess `bc` is used here because `VERSION_ID` might not be an integer?

Reply:
keep in mind that this will break if the minor version (the part after the first `.`) goes beyond 9

```
bc 1.07.1
Copyright 1991-1994, 1997, 1998, 2000, 2004, 2006, 2008, 2012-2017 Free Software Foundation, Inc.
This is free software with ABSOLUTELY NO WARRANTY.
For details type `warranty'. 
8.10 > 8.9
0
```

And it will also break if there is more than one `.`

```
8.10.1 > 8.9
(standard_in) 2: syntax error
```

Reply:
Yeah, it's about time we tidied that up since we don't list bs as a dependency and it quite often isn't included in container images it's a pain. We also don't need to check for FC < 22 any more either. See latest push and thanks @smithfarm